scripts/functional_gem_behavior.sh

```diff
@@ -232,24 +264,14 @@ else
   pass "It does."
 fi
 
-desc "rzo roles with no local .rizzo.json is expected to have no output"
+desc "rzo roles with no personal .rizzo.json is expected to warn"
 stdout=$(mktemp -t XXXXXX.stdout)
 stderr=$(mktemp -t XXXXXX.stderr)
 rzo roles 2> $stderr > $stdout
-if [[ -s $stderr ]]; then
-  echo "STDERR is expected to be empty, got:"
-  cat $stderr
-  fail "rzo roles produced STDERR output."
-else
-  pass "STDERR is zero bytes"
-fi
-if [[ -s $stdout ]]; then
-  echo "STDOUT is expected to be empty, got:"
-  cat $stdout
-  fail "rzo roles produced STDOUT output."
-else
-  pass "STDOUT is zero bytes"
-fi
+
```

Comment:
Since each schema validation has its own warnings, we should have an empty .rizzo.json with just `{}` and see that all the expected warnings happen.